Source/WebKit/UIProcess/WebPermissionControllerProxy.cpp

```diff
@@ -77,7 +77,7 @@ RefPtr<WebPageProxy> WebPermissionControllerProxy::mostReasonableWebPageProxy(co
             for (auto& potentialWebPageProxy : getPtr(process)->pages()) {
                 if (!potentialWebPageProxy)
                     continue;
-                if (WebCore::SecurityOriginData::fromURL(URL { potentialWebPageProxy->currentURL() }) != topOrigin)
+                if (!protocolHostAndPortAreEqual(URL { potentialWebPageProxy->currentURL() }, topOrigin.toURL()))
```

Comment:
There are quite a few places like this why I don't understand why we didn't simply use `SecurityOriginData::fromURLIgnoringOpaqueness()` instead of `protocolHostAndPortAreEqual()`.